rust/tests/integration/webrtc_transport.rs

```diff
@@ -161,6 +160,8 @@ fn create_succeeds() {
                         .try_into()
                         .unwrap(),
                     );
+                    // TODO: I wonder how this was not needed before...
```

Comment:
Nazar? `enable_sctp` must be true and it was not explicit before. I wonder how was it working before. I don't see what I'm missing.

Reply:
Not sure, was it set in TypeScript before?